ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/network/NetworkingServiceLifecycleTest.java

```diff
@@ -65,41 +66,61 @@ public void createP2PNetwork() throws IOException {
     }
   }
 
-  @Test(expected = IllegalArgumentException.class)
+  @Test
   public void createP2PNetwork_NullHost() throws IOException {
     final NetworkingConfiguration config =
         NetworkingConfiguration.create()
             .setDiscovery(DiscoveryConfiguration.create().setBindHost(null));
-    try (final P2PNetwork broken = builder().config(config).build()) {
-      Assertions.fail("Expected Exception");
-    }
+    final DefaultP2PNetwork.Builder p2pNetworkBuilder = builder().config(config);
+    assertThatThrownBy(
+            () -> {
+              try (final P2PNetwork ignored = p2pNetworkBuilder.build()) {
+                Assertions.fail("Expected Exception");
```

Comment:
do we still need this with assertThatThrownBy?

Reply:
No, we don't really need it, but if I remove that, Sonar complains about an empty block, and if I remove the _try with resources block_ then it complains about not handling the lifecycle of the closable object 🤪

Reply:
gotcha. all good